SDPi_Supplement/asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

```diff
@@ -0,0 +1,326 @@
+===== PID - Patient Identification Segment
+The HL7 Patient Identification (PID) segment requires a mapping from the SDC patient context information to the PID segment fields.
+
+.Prerequisite of Valid Patient Context
+[NOTE#ref_valid_pt_ctxt_note]
+====
+The SDC patient context information shall only be mapped to the corresponding fields in the HL7 PID segment when the requirements (e. g. inferred patient ensemble context, associated patient context, etc.) as defined in the <<ieee_11073_10700_2022>> are fulfilled.

```

Comment:
This should be transformed to an sdpi_requirement block.

Reply:
Done.